src/main/java/by/exadel/internship/mail/EmailTemplate.java

```diff
@@ -0,0 +1,7 @@
+package by.exadel.internship.mail;
+
+public class EmailTemplate {
+    public static final String FROM = "test.exadel.test@gmail.com";
+    public static final String SUBJECT = "Test Exadel";
+    public static final String TEXT = "Your form has been successfully submitted.";
```

Comment:
As I remember you would like to add name here?

Reply:
done